helmfile/values/opendistro-es.yaml.gotmpl

```diff
@@ -275,7 +275,11 @@ elasticsearch:
 
 
     ############## Elasticsearch configuration settings ##############
+    {{- if .Values.elasticsearch.indexPerNamespace }}
+    action.auto_create_index: "-kubeaudit,-authlog,-other,+*"
```

Comment:
I probably just don't understand this particular config. But I'm guessing this is some index pattern thing. So should you really have the `-` sign before all these? And are you creating one pattern for `+*`?

Reply:
`-` means dissallow, `+` means allow.

Fluentd should be be able to create `kubeaudit/other/authlog` indexes, remember it still writes to those aliases even with this new featre. It's not likely but can happen unless the configurer has gotten around to create the indices before fluentd starts to write to the aliases.

Reply:
And yeah maybe we could remove the `+*`, it didn't test without.